io/disk/bonnie.py

```diff
@@ -99,15 +108,16 @@ def test(self):
         args.append('-n %s' % self.number_to_stat)
         args.append('-s %s' % self.data_size)
         args.append('-u %s' % self.uid_to_use)
-
-        cmd = ('%s/bonnie++ %s' % (self.source, " ".join(args)))
+        cmd = ('bonnie++ %s' % " ".join(args))
         if process.system(cmd, shell=True, ignore_status=True):
             self.fail("test failed")
 
     def tearDown(self):
         '''
         Cleanup of disk used to perform this test
         '''
+        if os.listdir(self.source):
+            build.make(self.source, extra_args='clean')
```

Comment:
As it will not `uninstall` the binary from system, can you help me why it is needed?

Reply:
aah you are right i thought it will uninstall binary as well